metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Analysis.java

```diff
@@ -118,32 +118,37 @@
     @Column(nullable = false, columnDefinition = "TEXT")
     private String description;
 
-    @ApiModelProperty(position = 5, dataType = "java.lang.String", notes = "Url to a Study")
+    @ApiModelProperty(position = 5, dataType = "java.lang.String", notes = "Url to a Project")
+    @JsonProperty
+    @ManyToOne(optional = true) // TODO jmmut: I'm putting this as optional until we complete the project import from DB
+    private Project project;
```

Comment:
Since Project <1..1>* Study <1..M> Analysis, is there a need to explicitly fill the analysis-to-project link if we already have an analysis-to-study link? Can't the analysis-to-project be a computed property?

_\* Presumably, at least in our use case_

Reply:
It can be a computed property. I did this because EVA will want to easily link projects and analysis without knowing the studies, but the code can allow this transparently. I will add a ticket for this.